pkg/operation/botanist/monitoring.go

```diff
@@ -212,9 +209,6 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 			"nodeLocalDNS": map[string]interface{}{
 				"enabled": b.Shoot.NodeLocalDNSEnabled,
 			},
-			"reversedVPN": map[string]interface{}{
-				"enabled": b.Shoot.ReversedVPNEnabled,
-			},
```

Comment:
Drop https://github.com/gardener/gardener/blob/0459ab2f6740bd03d1f11ce218a1e1602442c013/charts/seed-monitoring/charts/core/charts/prometheus/templates/prometheus.yaml#L62-L64 and https://github.com/gardener/gardener/blob/20ac4c438b5458750e5ebd69dbb56ee3cea4ca2b/charts/seed-monitoring/charts/core/charts/prometheus/values.yaml#L27-L28?

Reply:
This is already removed, or do I miss something here?

Reply:
Right, nevermind :)